Javascript/q42.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>q42</title>
+
+  <style>
+    body {
+      font-family: Verdana, Geneva, Tahoma, sans-serif;
+    }
+
+    .flex {
+      display: flex;
+      flex-wrap: wrap;
+      width: 300px;
+      gap: 10px;
+    }
+
+    .box {
+      width: 200px;
+    }
+
+    button {
+      margin-top: 5px;
+    }
+  </style>
+
+  <script>
+    function myFunction(){
+      let min = Number(document.getElementById("demo0").value);
+      let max = Number(document.getElementById("demo1").value);
+      let x = Math.round(Math.random() * (max - min + 1) + min);
```

Comment:
What is `x`? Please improve this variable name

Reply:
done the changes